messages/run.js

```diff
@@ -28,7 +23,11 @@ module.exports = {
 		"vceDescription": "throws an error when violations are detected",
 		"vceDescriptionLong": "Throws an error when violations are detected. Exit code is the most severe violation.",
 		'engineDescription': "engine(s) to run",
-		'engineDescriptionLong': "One or more engines to run. Multiple values can be specified as a comma-separated list."
+		'engineDescriptionLong': "One or more engines to run. Multiple values can be specified as a comma-separated list.",
+		'eslintConfig': 'location of eslintrc config to customize eslint engine',
+		'eslintConfigLong': 'Location of eslintrc to customize eslint engine',
+		'pmdConfig': 'location of PMD rule reference XML file to customize rule selection',
+		'pmdConfigLong': 'Location of PMD rule reference XML file to customize rule selection'
```

Comment:
Could you change these to `xDescription` and `xDescriptionLong` so they match the established convention?

Reply:
Good call. Will fix this.